src/test/java/Homework21.java

```diff
@@ -1,10 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.time.Duration;
+
 public class Homework21 extends BaseTest {
-login("karkisuman0330@gmail.com","te$t$tudent");
-WebElement wait=null;
-@Test
+    Actions a = new Actions(driver);
+    WebDriverWait wait = null;
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/213907514-2e6aa5ee-c6e8-4ff2-b8f6-00050e839e1d.png)

Hint

![image](https://user-images.githubusercontent.com/104761482/213907772-3c524a0b-1d0e-45d0-8cca-3b3eb9bab02e.png)




![image](https://user-images.githubusercontent.com/104761482/213907811-e2f488a0-c25b-4776-a618-ab8fb9bd4560.png)